wiki/BBCode/de.md

```diff
@@ -0,0 +1,304 @@
+
+Beispiele für korrekte und inkorrekte Benutzungen von verschachtelten Tags ist unten näher beschrieben:
+
+- `[centre][b]Text[/b][/centre]` ist korrekt
+- `[b][centre]Text[/b][/centre]` ist inkorrekt
+
+## Tags
+
+BBCode, wie viele andere Markup-Sprachen, formatiert Text durch ein System von Tags, die durch ein Paar eckiger Klammern (`[]`) gekennzeichnet sind. Diese Tags sind aufgeteilt in "öffnende" und "schließende" Tags, die durch das Einfügen eines Schrägstrichs (`/`) unterscheidbar werden. D. h., dass schließende Tags einen Schrägstrich direkt nach der öffnenden Klammer haben, wohingegen öffnende Tags dies nacht haben.
+
+Es ist auch wichtig zu wissen, dass öffnende Tags manchmal Gleichheitszeichen (`=`) enthalten, um URLs, Schriftgröße und andere derartige Elemente zu markieren.
+
+BBCode-Tags, die von der osu!-Webseite unterstützt werden, sind unten aufgelistet und näher erläutert.
+
+### Fettschrift
+
+```
+[b]Text[/b]
+```
+
+Das Tag `[b]` wird benutzt, um Text mit Hilfe von Fettschrift stark zu betonen. Die Schriftgröße wird hierbei nicht verändert.
+
+Schaltfläche in der Symbolleiste: ![Button für Fettschrift](img/bold.png "Fett")
+
+### Kursivschrift
+
+```
+[i]Text[/i]
+```
+
+Das Tag `[i]` wird benutzt, um Text mit Hilfe einer Schräglage leicht zu betonen (d. h. in kursiv setzen).
+
+Schaltfläche in der Symbolleiste: ![Button für Kursivschrfit](img/italic.png "Kursiv")
+
+### Unterstreichen
+
+```
+[u]Text[/u]
+```
+
+Das Tag `[u]` wird benutzt, um mit Hilfe einer horizontalen Linie Text hervorzuheben (d. h. unterstreichen). Die horizontale Linie wird von anderen Tags beeinflusst, wie Fettschrift und Kursivschrift.
+
+### Durchstreichen
+
+```
+[strike]Text[/strike]
+```
+
+*Tipp: "strikethrough" ist auch bekannt als "strike".*
+
+Das Tag `[strike]` wird benutzt, um die Entfernung eines zuvor enthaltenen Text zu markieren. Die Kennzeichung erfolgt durch eine horizontale Linie, die den Text durchstreicht.
+
+Schaltfläche in der Symbolleiste: ![Button zum Durchstreichen](img/strike.png "Durchstreichen")
+
+### Farbe
+
+```
+[color=HEXCODE]Text[/color]
+```
+
+*Für eine Liste aller Farbkodierungen, siehe [X11 Farbkodierung](https://de.abcdef.wiki/wiki/X11_color_names)*
+
+Das Tag `[color]` wird benutzt, um Text mit unterschiedlichen Webfarben zu gestalten. Es wird das [Hexadezimalsystem](https://de.wikipedia.org/wiki/Webfarbe#RGB-Farbraum) verwendet, um die Farbe zu spezifizieren. Die Farbe kann aber auch durch HTML-Farbnamen, wie z. B. "red" oder "green" notiert werden. Um dem Text eine Farbe zu geben, ersetze das Argument `HEXCODE` mit einem Hexadezimal- oder HTML-Farbcode.
+
+Das zuvor genannte Argument darf keine Anführungszeichen (`"`) enthalten und hat keinen Standardwert. Wenn kein Wert festgelegt wurde oder, wenn Anführungszeichen benutzt werden, dann wird das Tag nicht als BBCode-Tag verarbeitet.
+
+### Schriftgröße
+
+```
+[size=NUMBER]Text[/size]
```

Comment:
Wouldn't it make sense to translate these indicators? I.e. use `NUMMER` instead of `NUMBER`.

Reply:
I wasn't sure about translation here, but sure. There's also `Address` further down in the article, so I will change that too.